Ahorn/lang/en_gb.lang

```diff
@@ -12,6 +12,10 @@ placements.entities.JungleHelper/ClimbableOneWayPlatform.tooltips.left=Whether t
 placements.entities.JungleHelper/ClimbableOneWayPlatform.tooltips.texture=Changes the appearance of the platform. Should be in the same format as vanilla for static jumpthrus, and a 8x8 image for animated ones.
 placements.entities.JungleHelper/ClimbableOneWayPlatform.tooltips.animationDelay=The delay in seconds between 2 frames of animation. Leave to 0 for no animation.
 placements.entities.JungleHelper/ClimbableOneWayPlatform.tooltips.surfaceIndex=Defines which climbing sounds the platform will use.
+placements.entities.JungleHelper/ClimbableOneWayPlatform.tooltips.staminaBehavior=Determines whether the player will conserve current stamina, regain stamina or exhibit normal behavior.
+placements.entities.JungleHelper/ClimbableOneWayPlatform.tooltips.sameDirectionJumpBoost=If this is checked and a player jumps across the platform, they will be given extra momentum as if they'd jumped from the other side.
+placements.entities.JungleHelper/ClimbableOneWayPlatform.tooltips.momentumJumpDecayTime=If a player lands on a platform with momentum and this is non-zero, that momentum will be stored and decay over time, reaching zero at the selected time. If a player jumps within this time, they are granted the respective stored speed. Must be >0 to store speed.
+placements.entities.JungleHelper/ClimbableOneWayPlatform.tooltips.momentumJumpDecayCurvature=If a player lands on a platform with momentum and this is non-zero, that momentum will be stored and decay over time following the curvature measure. Clamped bewteen 0 and 1. If closer to 1, speed will decay slower at first and decay faster over time. Approaching 0 will cause speed decay to approach a linear relation based on the decay time. Must be >0 to store speed.

```

Comment:
seems like 0.5 is what is close to linear 🤔

Reply:
Updated, ty for catching that!